src/Process.js

```diff
@@ -20,7 +20,7 @@ class Process {
     }
 
     isFinished() {
-
+      return this.cpuTimeNeeded >= 0 ? true : false;
```

Comment:
Looks like you just have the logic backwards on this one, which is probably why it's failing the test.

Also, using a ternary is not necessary since the expression alone is already going to return a true or false.